docs/source/adb.rst

```diff
@@ -0,0 +1,131 @@
+.. _adb:
+
+####################
+Android Debug Bridge
+####################
+
+.. important:: Android Debug Bridge is required ONLY for those users pairing an Android mobile device with Cozmo. iOS users require *usbmuxd*; instructions for installing this are given on the install pages for each operating system.
```

Comment:
I think wherever we use the full 'Android Debug Bridge' wording, we should instead say 'Android Debug Bridge (adb)'. The full words are very rarely used.